### ISSUE TITLE
Changing billing email doesn't work

### ISSUE THREAD

**Issue by saitheexplorer** [docker/hub-beta-feedback#290]
Tried to change my billing email for our organization's account - had to re-input my billing info to make this change. I got a 400 Bad Request on my first attempt. I don't have a secondary address line, so I made it ' ' (empty space) and it seemed to succeed, but the reloaded page showed me an account summary with the free plan selected (could be a bug or it could have been showing my personal account's details instead of my organization's).

Basically, I'm not able to change my billing email right now.

**Comment by nhsiehgit** [comment]
Currently there are issues with updating the billing account and even more so with updating an organizations billing account.

We've removed the ability to update your billing account for now but will work to re-implement this ability with the next deployment. If you need help changing this immediately, please contact our support team and they can help you update your account.

Sorry for the inconvenience and thanks for your patience!

**Comment by nhsiehgit** [comment]
closing since this closely duplicates: https://github.com/docker/hub-beta-feedback/issues/274
